src/components/Asset/AssetDetails.tsx

```diff
@@ -18,87 +16,73 @@ export type AssetDetailsProps = {
 }
 
 export const AssetSymbol: FC<AssetDetailsProps> = ({ address, isLoading }) => {
-  const isReady = useClientReady()
   const { data: token, isLoading: isLoadingToken } = useTokenOrNative({
     address,
   })
   return (
-    <Skeleton isLoading={isLoading || isLoadingToken || !isReady}>
-      {isReady
-        ? address
-          ? token?.symbol ?? "Loading..."
-          : "???"
-        : "Loading..."}
+    <Skeleton isLoading={isLoading || isLoadingToken}>
+      {address ? token?.symbol ?? "Unknown" : "???"}
     </Skeleton>
   )
 }
 
 export const AssetName: FC<AssetDetailsProps> = ({ address, isLoading }) => {
-  const isReady = useClientReady()
   const { data: token, isLoading: isLoadingToken } = useTokenOrNative({
     address,
   })
   const re = /(\(arb1\))/gi
   const tokenName = token?.name ?? "Loading..."
   const updatedTokenName = tokenName.replace(re, "")
   return (
-    <Skeleton isLoading={isLoading || isLoadingToken || !isReady}>
-      {isReady ? (address ? updatedTokenName : "Unknown token") : "Loading..."}
+    <Skeleton isLoading={isLoading || isLoadingToken}>
+      {address ? updatedTokenName : "Unknown token"}
     </Skeleton>
   )
 }
 
-type AssetBalanceProps = AssetDetailsProps &
-  Partial<Pick<CurrencyProps, "abbreviate">>
-
-type AssetBalanceUsdProps = AssetBalanceProps & {
-  asset?: Address | undefined
+type AssetBalanceProps = AssetDetailsProps & {
+  abbreviate?: boolean
 }
 
 export const AssetBalance: FC<AssetBalanceProps> = ({
   address,
   abbreviate,
 }) => {
-  const isReady = useClientReady()
   const { isConnected } = useAccount()
   const { data: balance, isLoading } = useTokenOrNativeBalance({ address })
   return (
-    <Skeleton isLoading={isLoading || !isReady}>
-      {!isReady || !isConnected || !address ? (
-        <>—</>
-      ) : (
-        <Currency
-          amount={balance?.value ?? BigNumber.from(0)}
-          decimals={balance?.decimals ?? 18}
-          abbreviate={abbreviate}
-        />
-      )}
+    <Skeleton isLoading={isLoading}>
```

Comment:
Do we no longer display "-" for non-connected users?

Reply:
We do its right there on line 61